Content.Server/CriminalRecords/Systems/CriminalRecordsConsoleSystem.cs

```diff
@@ -138,6 +140,8 @@ private void OnChangeStatus(Entity<CriminalRecordsConsoleComponent> ent, ref Cri
             (SecurityStatus.Detained, SecurityStatus.None) => "released",
             // criminal is no longer on parole
             (SecurityStatus.Paroled, SecurityStatus.None) => "not-parole",
+            // # DeltaV - Person is no longer under subpoena
```

Comment:
```suggestion
            // # DeltaV - Person is no longer subpoenaed
```
"under subpoena" is strange language.